### ISSUE TITLE
[Shaman][Spell] Double Windfury Weapon proccrate still too low

### ISSUE THREAD

**Issue by quapuzi** [Atlantiss/NetherwingBugtracker#6218]
**Description**:  Using Double Windfury Weapon instead of one doesnt increase the proccrate.

**Current behaviour**:
https://github.com/Atlantiss/NetherwingBugtracker/issues/6209 As explained here, using Windfury Weapon two times still leaves the proccrate the same. Proof:
https://legacyplayers.com/Raids/Viewer/Default.aspx?id=8455&exp=1&upl=12685
Looking at Rage Winterchill, Uhh has 167 Autohits landing and 2x 19 Stormstrike hits, means there are in total 205 Events for possible Windfury hits.
There are 66 Windfury hits in total. Since Windfury Weapon hits twice, this results in 33 Proccs
This results in a proccchance of:
(33/205)*100= ~16.1% Chance.
For using Windfury Weapon once this is right. Looking at the same log, same Shaman for Azgalor:
337 Autohits, of wich are ~50% Offhand (Flametongue on offhand) this results in about 168 Mainhand hits landing.
Now add 32 Stormstrikes on top of that, you have in total 200 Mainhand hits that can cause Windfury proccs.
There are 60 Windfury hits recorded in total. Again, Windfury weapon proccs twice, we have 30 Proccs.
(30/200)*100=15%
Since theres some rng (dodge/miss,swingtimer etc), is the proccrate exacle the same.

**Expected behaviour**: 
Using Windfury Weapon 2 Times has a hidden mechanic of increasing the proccchance from 20% to 36%, wich causes the rate to increase from 16% to 20%

Pointing out some quotes from previous report:
https://web.archive.org/web/20080811214521/http://elitistjerks.com/f31/t22760-enhancement_shaman_yo_help_me/
Post of • Toots Hepcat:

The reason 2x WF is superior to 1x WF, even though some of your WF attacks are smaller offhand attacks, is simple: when you have WF on one hand, the chance for that one hand to swing is 20%. However, when you put WF on BOTH hands, EACH hand has its proc chance increased to 36%.

TLDR:  Windfury now prefers Mainhand exactly like it should, just missing the hidden procchance increase.
**Server Revision**: 
3106

**Comment by Easyfraggy** [comment]
way too little data with 167 hits - need ptr updated to test properly.

**Comment by Godmatik** [comment]
![wf1](https://github.com/user-attachments/assets/f655937d-3d46-4258-b3eb-04354175011e)
![wf2](https://github.com/user-attachments/assets/e4c3982b-2761-4bde-a26d-52ba0a5d3b32)
![wf3](https://github.com/user-attachments/assets/8062ea51-3cd0-49f1-b3f8-c98d4eb376a9)
![wf4](https://github.com/user-attachments/assets/70b208c5-e19f-4141-91e8-53990a53d608)

Tested extensively - and found that Windfury proc rate is correct.

Also note that WF has a 3 second cooldown, so if your weapons are faster than that, the proc rate will seem lower.

Closing
